src/main/java/ca/corefacility/bioinformatics/irida/ria/web/samples/SamplesAjaxController.java

```diff
@@ -189,6 +189,7 @@ private void createSequenceFileInSample(MultipartFile file, Sample sample) throw
 	 */
 	private void createFast5FileInSample(MultipartFile file, Sample sample) throws IOException {
 		SequenceFile sequenceFile = createSequenceFile(file);
+		Fast5Object fast5Object = new Fast5Object(sequenceFile);
```

Comment:
This line isn't used.

Reply:
Removed in 56ac519f9ce8049dfaf894978357291373d24e8a